hashmap.js

```diff
@@ -166,11 +166,14 @@
 		}
 	};
 
+	// ES6 Map API compatibility

```

Comment:
The comment is not correct. The API is now compatible, this line of code is rather to keep backwards compatibility with the current codebase